client/src/components/SignUp.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { useState } from "react";
+
+import { Grid, Paper, Avatar, TextField, Button, Box } from "@material-ui/core";
+import CircleOutlined from "@material-ui/icons/AddCircleOutlineOutlined";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+
+interface SignUpProps {
+  handleChange: Function;
```

Comment:
Any way to do this without defining the interface? Not sure if we should have one if it is only used in one spot